controllers/druid/handler.go

```diff
@@ -705,21 +644,16 @@ func sdkCreateOrUpdateAsNeeded(
 		if err := sdk.Get(context.TODO(), *namespacedName(obj.GetName(), obj.GetNamespace()), prevObj); err != nil {
```

Comment:
Reason for not having called the get interfaces is cause GET shall log when it does not find the object, here the case is such that in case object not found create it and then log the error. So i ignored adding the writer.Get().